Doc/library/test.rst

```diff
@@ -1225,15 +1242,17 @@ The :mod:`test.support.threading_helper` module provides support for threading t
    is still alive after *timeout* seconds.
 
 
-.. decorator:: reap_threads(func)
+.. decorator:: reap_threads
 
    Decorator to ensure the threads are cleaned up even if the test fails.
 
 
 .. function:: start_threads(threads, unlock=None)
 
-   Context manager to start *threads*.  It attempts to join the threads upon
-   exit.
+   Context manager to start *threads*, which is a sequence of threads.
+   *unlock* is a function called after the threads are started, even if an
+   exception was raised; an example would be :meth:`threading.Event.set`.
+   This also attempts to join the started threads upon exit.
```

Comment:
Unclear whether "this" refers to ``start_threads`` or ``threading.Event.set```

```suggestion
   ``start_threads`` will attempt to join the started threads upon exit.
```